docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
 
 import RPC from "./../../components/RPC-Connection"
 
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements NPoS (Nominated
-Proof-of-Stake), a relatively novel and sophisticated mechanism to select the validators who are
-allowed to participate in its [consensus](learn-consensus.md) protocol. The NPoS encourages
-{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} holders to participate as nominators.
+Here you will learn about what staking is, why it is important and how it works on
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}.
+
+## Introduction to Proof-of-Stake (PoS)
+
+Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to finalize
+blocks on the chain. Consensus is the process of agreeing on something, in this case, the progression
+of the blockchain or how blocks are added to the chain. Consensus consists of two actions:
+
+- `block production`, i.e. the way multiple blocks candidates are produced, and 
+- `block finality`, i.e. the way only one block out of many candidates is selected and added to the canonical chain (see [this](learn-consensus.md/#probabilistic-vs-provable-finality) article for more information about finality).
+
+Proof-of-Work (PoW) and Proof-of-Stake (PoS) are well-known mechanisms used to reach consensus in a secure and trustless way on public blockchains, where there are many participants who do not know each other (and probably never will). In PoW, network security relies on the fact that the miners who are responsible for adding blocks to the chain must compete to solve difficult mathematic puzzles to add blocks - a solution that has been criticized for the wastage of energy. For doing this work, miners are typically rewarded with tokens. In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
```

Comment:
one paragraph for each would be a cleaner description, and I generally think both can be a bit more concise (but that subjective).

Reply:
Suggestion accpeted